dashboard/test/controllers/courses_controller_test.rb

```diff
@@ -81,13 +81,17 @@ class CachedQueryCounts < ActionController::TestCase
 
     test 'student views course overview with caching enabled' do
       sign_in create(:student)
+      populate_cache
```

Comment:
disconnecting the db here caused an error so I split out the populating of the cache into its own thing